src/event.js

```diff
@@ -626,8 +631,9 @@ jQuery.event = {
 };
 
 jQuery.removeEvent = function( elem, type, handle ) {
+	// This "if" is needed for plain objects
```

Comment:
So <a href="https://contribute.jquery.org/style-guide/js/#comments">this</a> - 
```js
function foo( types, selector, data, fn, /* INTERNAL */ one ) {
    // Do stuff
}
```
Is a mistake in guide?